src/modules/task.js

```diff
@@ -0,0 +1,7 @@
+export default class Task {
+  constructor(index, completed, description) {
+    this.index = index;
+    this.completed = completed;
```

Comment:
## [OPTIONAL] i mark this optional because it was not mentioned by the previous reviewer 
* I strongly suggest you set the property index to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). HINT 💭 you can change line `3` of your code to this `this.index = index + 1` this way when you create a todo list, the index starts from `1` instead of `0`

Reply:
I notice you in the reviews, brother. You give the best reviews I've ever seen. Thank u so much, my brother.